lightwood/encoders/time_series/rnn.py

```diff
@@ -81,23 +97,31 @@ def prepare(self, priming_data, feedback_hoop_function=None, batch_size=256):
 
             while data_idx < len(priming_data):
```

Comment:
Ok, a bit off-topic, but isn't it easy enough to use a DataLoader for this instead of manually doing batching :?

Reply:
Yeah, I agree. We should aim for that in a new PR.